src/pages/work-orders/[id]/choose-option.js

```diff
@@ -0,0 +1,20 @@
+import ChooseOption from '../../../components/WorkOrders/ChooseOption'
+import { CONTRACTOR_ROLE, CONTRACT_MANAGER_ROLE } from '../../../utils/user'
+
+const ChooseWorkOrderOption = ({ query }) => {
```

Comment:
How about naming it as `WorkOrderChooseOption`.

By doing that means we will start to follow a pattern. Having `WorkOrder` first as the resource and `ChooseOption` as the action.

Reply:
* `WorkOrderChooseOptionPage` if we are having the `Page` at the end of all other files